org.mixedrealitytoolkit.input/Assets/Input Actions/MRTK Default Input Actions.inputactions

```diff
@@ -1103,6 +1103,33 @@
                     "processors": "",
                     "interactions": "",
                     "initialStateCheck": true
+                },
+                {
```

Comment:
@ms-RistoRK  this actually doesn't work 

The problem is that when eye's are calibrated the eyes are still available and unity tries to obtain data from them.  I needed to write the "fallback" script to check the tracking state of the eyes, and then fallback to the head pose.

Perhaps XRI3 is different. but sort of doubt it.

Reply:
Got it!, thx for the clarification.  I'll retry the original approach then.  New approach reverted in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/746/commits/3ea4039f4cc9eea4de80226a30141eddce3d7308 , resolving.